go/saltpack/common.go

```diff
@@ -45,3 +47,8 @@ func hashNonceAndAuthTag(nonce *Nonce, ciphertext []byte) []byte {
 	buf.Write(ciphertext[0:poly1305.TagSize])
 	return buf.Bytes()
 }
+
+func writeNullString(w io.Writer, s string) {
```

Comment:
can you rename this to `writeNullTerminatedString`?  otherwise, i'd think it's a function that just writes `[]byte{0}` and nothing else.